dockers/docker-fpm-frr/frr/bgpd/templates/voq_chassis/policies.conf.j2

```diff
@@ -12,6 +13,17 @@ route-map FROM_VOQ_CHASSIS_V4_PEER permit 1
 route-map FROM_VOQ_CHASSIS_V4_PEER permit 2
   match community NO_EXPORT
   set local-preference 80
+  on-match next
+!
+{% if CONFIG_DB__DEVICE_METADATA['localhost']['subtype'] == 'UpstreamLC' %}
```

Comment:
is this correct, should it be `DownstreamLC` ?

Reply:
yes, this condition is to check if any azng route (eligible for fallback)  learnt via iBGP on upstream LC should be deny

Reply:
> yes, this condition is to check if any azng route (eligible for fallback) learnt via iBGP on upstream LC should be deny